backend/endpoints/messages.py

```diff
@@ -143,18 +194,13 @@ async def update_message(
     )
 
     if edited_message and edited_message.get("status_code") is None:
-        new_data = {
-            "room_id": room_id,
-            "message_id": message_id,
-            "sender_id": payload["sender_id"],
-            "text": payload["text"],
-            "edited": True,
-        }
+        new_message = await get_message(org_id, room_id, message_id)
```

Comment:
Why do we need to get the message back?

Reply:
That's what Aniebet requested yesterday. He said he wanted the entire object to be returned and published to centrifugo.

Reply:
@man-of-mind That was actually requested by Aniebet. He said that the entire updated object should be returned and published to centrifugo instead of just the updated payload.

Reply:
Sorted boss